npm_and_yarn/lib/dependabot/npm_and_yarn/metadata_finder.rb

```diff
@@ -39,7 +39,7 @@ def look_up_source
 
         case source_type
         when "git" then find_source_from_git_url
-        when "private_registry" then find_source_from_registry
+        when "registry" then find_source_from_registry
```

Comment:
I think we only run this during the update now while creating the pull request so shouldn't run in backend with old attribute 👌